src/components/Loading.jsx

```diff
@@ -0,0 +1,13 @@
+import styles from '../styles/loading'
+
+import React from 'react'
+
+const Loading = ({ message }) => {
+  return (
+    <div className={styles['fil-loading']}>
+      <p>{message}</p>
+    </div>
+  )
+}
+
+export default Loading
```

Comment:
I know, right? Sometime earlier, I asked to keep *dumb component* out of third libraries like `i18n.js`. We told me it's better to have a dumb component requesting keys instead of fully translated sentences. I think it's wrong, we may have *dumb component* requesting strings instead of i18n keys:

```jsx
const myDumbComponent = ({ title, subtitle, paragraph }) => (
  <section>
    <h1>{title}</h1>
    <h2>{subtitle}</h2>
    <p>{paragraph}</p>
  </section>
)
```

instead of a strongly coupled component:

```jsx
const myNotReallyDumbComponent = ({ t, titleKey, subtitleKey, paragraphKey }) => (
  <section>
    <h1>{t(titleKey)}</h1>
    <h2>{t(subtitleKey)}</h2>
    <p>{t(paragraphKey)}</p>
  </section>
)
```